### PR TITLE
SKALE-2972 Move turn on/off scripts to python, move iptables

### PR DESCRIPTION
**Pull request by dmytrotkk**

### CODE REVIEW

File: node_cli/core/node.py [hunk 1]
```diff
@@ -116,8 +115,13 @@ def restore(backup_path, env_filepath):
     if env_params is None:
```
Comment (badrogger) [inline node_cli/core/node.py]: I think it's better to use get_node_env here.

File: node_cli/operations/common.py [hunk 2]
```diff
@@ -82,11 +80,8 @@ def configure_flask():
...
 
-def configure_iptables():
-    try:
-        run_cmd(['bash', CONFIGURE_IPTABLES_SCRIPT])
-    except Exception:
-        error_msg = 'iptables configure script failed'
-        logger.exception(error_msg)
-        error_exit(error_msg, exit_code=CLIExitCodes.SCRIPT_EXECUTION_ERROR)
-    logger.info('iptables configured successfully')
+def unpack_backup_archive(backup_path: str) -> None:
```
Comment (badrogger) [inline node_cli/operations/common.py:83]: It's possible to use context manager here. Let's do it.

File: node_cli/core/iptables.py [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+
+def configure_iptables():
+    """
+    This is the main function used for the initial setup of the firewall rules on the SKALE Node
+    host machine
+    """
+    logger.info('Configuring iptables...')
+    if not iptc:
+        raise Exception('Unable to import iptc package')
```
Comment (badrogger) [inline node_cli/core/iptables.py]: Let's raise custom exception

**Review by badrogger — CHANGES_REQUESTED** [review]

File: node_cli/core/node.py (hunk 1 above)
Comment (dmytrotkk) [reply node_cli/core/node.py]: replaced

File: node_cli/operations/common.py (hunk 2 above)
Comment (dmytrotkk) [reply node_cli/operations/common.py:83]: done

File: node_cli/core/iptables.py (hunk 3 above)
Comment (dmytrotkk) [reply node_cli/core/iptables.py]: changed to `ImportError`

**Review by badrogger — APPROVED** [review]

**Pull request merged**